src/main/java/bio/terra/service/resourcemanagement/google/GoogleBucketService.java

```diff
@@ -277,12 +277,15 @@ private Bucket newCloudBucket(GoogleBucketResource bucketResource) {
     boolean doVersioning =
         Arrays.stream(env.getActiveProfiles()).noneMatch(env -> env.contains("test"));
     String bucketName = bucketResource.getName();
+    GoogleRegion region = bucketResource.getRegion();
+    StorageClass storageClass =
+        region.isMultiRegional() ? StorageClass.MULTI_REGIONAL : StorageClass.REGIONAL;
     BucketInfo bucketInfo =
         BucketInfo.newBuilder(bucketName)
             // .setRequesterPays()
             // See here for possible values: http://g.co/cloud/storage/docs/storage-classes
-            .setStorageClass(StorageClass.REGIONAL)
-            .setLocation(bucketResource.getRegion().getRegionOrFallbackBucketRegion().toString())
+            .setStorageClass(storageClass)
+            .setLocation(region.toString())
```

Comment:
is there a way that we could have a simple connected or integration test to make sure this behaves as expected?

Reply:
Yes, but I'd like to get this out ASAP since it's blocking HCA on stuff.

Reply:
Added a test and it passes locally.